applications/app/views/fragments/newGalleryBody.scala.html

```diff
@@ -0,0 +1,127 @@
+@(gallery: model.Gallery, related: model.RelatedContent, index: Int)(implicit request: RequestHeader)
+
+@import common.{Edition, LinkTo}
+@import layout.ContentWidths.GalleryMedia
+@import views.support.TrailCssClasses.toneClass
+@import views.support.{ImgSrc, RenderClasses}
+
+<div class="l-side-margins l-side-margins--media l-side-margins--new-gallery">
+
+    <article id="article" class="@RenderClasses(
+            Map("content--advertisement-feature" -> gallery.commercial.isAdvertisementFeature,
+                "paid-content--advertisement-feature" -> gallery.commercial.isAdvertisementFeature
+            ),
+            "content", "content--media", "content--gallery", "tonal", "new-gallery", "content--immersive", s"tonal--${toneClass(gallery)}"
+        )"
+        itemscope itemtype="@gallery.metadata.schemaType" role="main">
+
+        @if(gallery.commercial.isAdvertisementFeature){
+            @fragments.guBand()
+        }
+
+        @fragments.galleryHeader(gallery, gallery.commercial.isSponsored(Some(Edition(request))) || gallery.commercial.isFoundationSupported || gallery.commercial.isAdvertisementFeature)
+
+        <div class="content__main tonal__main tonal__main--@toneClass(gallery)">
+            <div class="gs-container">
```

Comment:
does this div still need to be a wrapper? could `gs-container` go on the div on line 26?